snuba/web/rpc/timeseries.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+from typing import Any
+
+from snuba.protobufs.base_messages_pb2 import (
+    AggregationType,
+    Comparison,
+    PentityAggregation,
+    PentityFilter,
+    PentityFilters,
+    RequestInfo,
+)
+from snuba.protobufs.time_series_pb2 import TimeSeriesRequest
+from snuba.utils.metrics.timer import Timer
+from snuba.web.query import parse_and_run_query
+
+# TODO: generate
+# , TimeSeriesResponse
+
+TimeSeriesResponse = Any
+
+
+def _agg_type_to_ch_function(agg_type: AggregationType) -> str:
+    lookup = {
+        AggregationType.AVG: "avg",
+        AggregationType.MIN: "min",
+        AggregationType.MAX: "max",
+        AggregationType.P50: "quantiles(0.5)",
```

Comment:
Can you change this to `quantile` instead? Otherwise you'll get an array in the results.

Reply:
Thank you!